gammapy/makers/spectrum.py

```diff
@@ -57,6 +57,9 @@ def make_exposure(self, geom, observation):
         exposure = super().make_exposure(geom, observation, use_region_center=self.use_region_center)
 
         if self.containment_correction:
+            if observation.aeff.is_pointlike:
```

Comment:
Note the access to `observation.aeff` reloads the IRF from disk, to avoid this, maybe work with a local variable. I also think we should keep the meta info as much as possible. So maybe we copy over the meta data to the reduced maps?
So you could access e.g.`exposure.meta["is_pointlike"]` here...